lib/active_shipping/shipment_event.rb

```diff
@@ -2,14 +2,18 @@ module ActiveShipping
   class ShipmentEvent
     attr_reader :name, :time, :location, :message, :type_code
 
-    def initialize(name, time, location, message = nil, type_code = nil)
-      @name, @time, @location, @message, @type_code = name, time, location, message, type_code
+    def initialize(name, time, location, message = nil, type_code = nil, timezone_ambiguous = true)
```

Comment:
[USPS is ambiguous.](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/carriers/usps.rb#L263)
[UPS as well.](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/carriers/ups.rb#L874)

Worth auditing them all though. I think Canada Post _sometimes_ gives the timezone.